File: Python/2 - registro.py
print("A continuacion podrá agregar datos personales")
registros = []
while True:
    registro = {}
    nombre = input("Introduzca el nombre: ")
    apellido = input("Introduzca el apellido: ")
    cedula = input("Introduzca la cedula: ")
    correo = input("Introduzca el correo electronico: ")
    telefono = input("Introduzca el telefono: ")

    registro = {
        "nombre": nombre, 
         "apellido": apellido, 
         "cedula":cedula, 
         "correo electrónico":correo, 
         "teléfono":telefono}
    
    registros.append(registro)

    salir = input("Si desea salir presione s, para continuar presione cualquier tecla: ")

    if salir == 's' or salir == 'S':
        print("Saliendo del programa.")
        break

for registro in registros:
    print("------------------")
    for clave, valor in registro.items():
        print(f"{clave} = {valor}")

File: Julia/2 - registro.jl
println("A continuacion podra agregar datos personales")
registros = []
while true
    println("Introduzca el nombre: ")
    nombre = readline()
    println("Introduzca el apellido: ")
    apellido = readline()
    println("Introduzca la cedula: ")
    cedula = readline()
    println("Introduzca el correo electronico: ")
    correo = readline()
    println("Introduzca el telefono: ")
    telefono = readline()

    registro = Dict(
        "nombre" => nombre,
        "apellido" => apellido,
        "cedula" => cedula,
        "correo" => correo,
        "telefono" => telefono
    )
    push!(registros, registro)

    println("Si desea salir, presione s, para continuar presione cualquier tecla: ")
    salir = readline()

    if salir == "s" || salir == "S"
        print("Saliendo del programa.\n")
        break
    end
end

for registro in registros
    print("------------------\n")
    for (clave, valor) in registro
        println("$clave = $valor")
    end
end